client/lib/plugins/sanitize-section-content.js

```diff
@@ -103,12 +127,13 @@ const isValidYoutubeEmbed = ( node ) => {
 		return false;
 	}
 
-	const link = document.createElement( 'a' );
+	const link = root.document.createElement( 'a' );
 	link.href = node.getAttribute( 'src' );
+	const url = new URL( link.href );
```

Comment:
This was needed because the server-friendly parser cannot construct the URL object within the `a` element.

Reply:
Probably want a try catch around this just to be safe. Relative paths in any links will throw here.

Reply:
Thanks for catching that! I pushed a fix.